### PR TITLE
Handle Chromedriver arguments

### PR DESCRIPTION
**Pull request by imurchie**
Strip the `goog` prefix off `chromeOptions` if it is present. And shift `Arguments` over to `args`.

See https://github.com/appium/appium/issues/9838

### CODE REVIEW

File: lib/commands/context.js [hunk 1]
```diff
@@ -211,6 +211,11 @@ async function setupNewChromedriver (opts, curDeviceId, adb) {
     verbose: !!opts.showChromedriverLog,
   };
   let chromedriver = new Chromedriver(chromeArgs);
+
+  // strip `goog` prefix if it is given
+  if (opts['goog:chromeOptions']) {
```
Comment (mykola-mokhnach) [inline lib/commands/context.js]: I'd rather accept everything, which matches `[\w-]+:chromeOptions` pattern

File: lib/webview-helpers.js [hunk 2]
```diff
@@ -132,6 +132,11 @@ helpers.getWebviews = async function (adb, deviceSocket) {
 helpers.decorateChromeOptions = function (caps, opts, deviceId) {
   // add options from appium session caps
   if (opts.chromeOptions) {
+    if (opts.chromeOptions.Arguments) {
+      // merge `Arguments` and `args`
+      opts.chromeOptions.args = (opts.chromeOptions.args || []).concat(...opts.chromeOptions.Arguments);
```
Comment (mykola-mokhnach) [inline lib/webview-helpers.js]: `opts.chromeOptions.args = [...opts.chromeOptions.args, ...opts.chromeOptions.Arguments]`

**Comment by imurchie** [comment]
Updated: remove _any_ prefix from `chromeOptions`, and merge.

File: lib/commands/context.js [hunk 3]
```diff
@@ -211,7 +211,19 @@ async function setupNewChromedriver (opts, curDeviceId, adb) {
...
   };
   let chromedriver = new Chromedriver(chromeArgs);
-  let appPackage = (opts.chromeOptions && opts.chromeOptions.androidPackage) || opts.appPackage;
+
+  // make sure there are chromeOptions
+  opts.chromeOptions = opts.chromeOptions || {};
+  // try out any prefixed chromeOptions,
+  // and strip the prefix
+  for (let opt of _.keys(opts)) {
+    if (opt.includes(':chromeOptions')) {
```
Comment (mykola-mokhnach) [inline lib/commands/context.js]: perhaps endsWith?

**Pull request merged**